test/e2e/system_reset_test.go

```diff
@@ -97,4 +98,33 @@ var _ = Describe("podman system reset", func() {
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToStringArray()).To(BeEmpty())
 	})
+
+	It("podman system reset, check podman.service", func() {
```

Comment:
I am not sure if a e2e test is good for this. All tests run with different --root --runrooot. Calling systemctl stop on podman.service seems wrong in such cases. We have no control on what root/runroot are used in the service file.

Reply:
same comment as above: how many use cases are we aiming to support within `podman system reset`